public/pages/DetectorsList/utils/helpers.ts

```diff
@@ -56,3 +58,33 @@ export const getDetectorStateOptions = () => {
     text: detectorState,
   }));
 };
+
+export const getValidDetectors = (
```

Comment:
maybe you can name it like `getDetectorsForAction`

Reply:
that's a better name, will change